auth/iomadoidc/manageapplication.php

```diff
@@ -72,6 +70,13 @@
     }
 }
 
+// Fix missing Certificate auth method if set in database, needs to be done before form is created
+if (isset($formdata['idptype'])) {
+    $iomadoidcconfig->idptype = $formdata['idptype'];
```

Comment:
If I'm reading this correctly, `isset($formdata['idptype']` is true if `isset($iomadoidcconfig->idptype)` or `isset($iomadoidcconfig->idptype_$companyid)`. Therefore is this ensuring that `$iomadoidcconfig->idptype` is set whether `$companyid` is set or not? If so, could the if statement be replaced by something like `$idptype = 'idptype' . $postfix; $iomadoidcconfig->idptype = $iomadoidcconfig->$idptype` or I missing something?

Reply:
Hi @Fydon sorry I must have missed your comment from a while back, the reason I chose to not modify the definition too much (eg being Company ID aware) is because this file is taken from the Moodle auth_oidc plugin and is not an IOMAD file, specifically the code from here: https://github.com/microsoft/o365-moodle/blob/master/auth/oidc/classes/form/application.php

I figured it would be best to not introduce drastic changes and to keep it as a 'translation layer' of sorts